### PR TITLE
FAT-3031 Implement karate test to get the  current module id for age-to-lost processor delay time

### PR DESCRIPTION
**Pull request by MurodjonMukhamadaliev**
https://issues.folio.org/browse/FAT-3031

 Issue was - problem with  getting suitable module id for age-to-lost processor delay time. If module id changes it was not possible to get the latest module id.

Solution: Implemented karate test that makes possible to get latest module id!

### CODE REVIEW

File: mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature [hunk 1]
```diff
@@ -933,21 +933,17 @@ Feature: Loans tests
     * def checkOutResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: #(extUserBarcode), extCheckOutItemBarcode: #(extItemBarcode), extLoanDate: #(extLoanDate) }
     * def extLoanId = checkOutResponse.response.id
 
-    # find current module id for age-to-lost processor delay time
+       # get the  current module id for age-to-lost processor delay time
```
Comment (roman-barannyk) [inline mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature:936]:
```suggestion
  # get the  current module id for age-to-lost processor delay time
```

File: mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature [hunk 2]
```diff
@@ -933,21 +933,17 @@ Feature: Loans tests
...
     Given path '/_/proxy/tenants/' + tenant + '/timers'
     When method GET
     Then status 200
-    * def fun = function(module) { return module.routingEntry.pathPattern == '/circulation/scheduled-age-to-lost' }
-    * def modules = karate.filter(response, fun)
-    * def currentModuleId = modules[0].id
+    * def fun = function(condition) {return condition.routingEntry.pathPattern == "/circulation/scheduled-age-to-lost" }
```
Comment (roman-barannyk) [inline mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature:940]:
```suggestion
    * def fun = function(condition) {return condition.routingEntry.pathPattern == '/circulation/scheduled-age-to-lost' }
```

File: mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature [hunk 3]
```diff
@@ -933,21 +933,17 @@ Feature: Loans tests
...
+    * def condition =  karate.filter(response, fun)
```
Comment (roman-barannyk) [inline mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature:941]:
```suggestion
    * def condition = karate.filter(response, fun)
```

**Comment by MurodjonMukhamadaliev** [comment]
not necessary

**Pull request closed without merging**